cli_tools/gce_vm_image_import/importer/bootable_disk_processor_test.go

```diff
@@ -16,25 +16,41 @@ package importer
 
 import (
 	"bytes"
+	"fmt"
 	"os"
 	"testing"
 
 	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestBootableDiskProcessor_Process(t *testing.T) {
+	for i, tt := range uefiTests {
+		name := fmt.Sprintf("%v. inspect disk: disk is UEFI: %v, input arg UEFI compatible: %v", i+1, tt.isUEFIDisk, tt.isInputArgUEFICompatible)
```

Comment:
Do we need all four of the test cases?

Reply:
2 is enough for this processor. let me remove the dup

Reply:
In bootable_disk_processor, I'm not logic that differentiates between whether the disk has an EFI partition. Can you point me to that?